_breadcrumbs.scss

```diff
@@ -0,0 +1,116 @@
+// Breadcrumbs
+// ===
+
+
+// Configurable Variables
+// ---
+
+$breadcrumbs__divider-content: '>' !default; // TO DO: include stencil icon
+$breadcrumbs__show-more-content: '...' !default; // TO DO: include stencil icon
+
+
+// Dependencies
+// ---
+
+@import 'stencil-variables/variables';
+@import 'spline/dist/utilities/hide-text';
+@import 'spline/dist/utilities/icon-font-replace';
+@import 'level/level';
+
+
+// Breadcrumbs
+// ---
+//
+
+.c-breadcrumbs {
+    display: flex;
+    overflow: auto;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+
+    -webkit-overflow-scrolling: touch;
+}
+
+
+// Breadcrumbs item
+// ---
+//
+// 1. Vertical align text center
+// 2. Hide breacrumb item with hidden attribute
+
+.c-breadcrumbs__item {
+    display: flex; // 1
+    align-items: center; // 1
+    flex: 0 0 auto;
+
+    &[hidden] {
+        display: none; // 2
+    }
+
+    // Active style
+    // ---
+    //
+
+    &.c--active {
+
+        .c-breadcrumbs__link {
+            font-weight: bold;
+        }
+    }
+
+    // Breadcrumbs divider
+    // ---
+    //
+    // 1. Divider content
+
+    + .c-breadcrumbs__item {
+        &::before {
+            content: $breadcrumbs__divider-content; // 1
+
+            display: inline-block;
+        }
+    }
+}
+
+
+// Breadcrumbs link
+// ---
+//
+
+.c-breadcrumbs__link {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
```

Comment:
Seeing height set to me is always a risk. In this case, it makes sense, since the container allows for scrolling and the content won’t wrap or be permanently obscured.